docs/quickstart.md

```diff
@@ -0,0 +1,347 @@
+
+https://github.com/openebs/zfs-localpv/blob/HEAD/docs/faq.md#6-how-to-add-custom-topology-key
+
+<BR>
+
+## Installation
+In order to support moving data to a new node later on, you must label each node with a unique value for `openebs.io/nodeid`.
+For more information on migrating data, please [see here](docs/faq.md#8-how-to-migrate-pvs-to-the-new-node-in-case-old-node-is-not-accessible)
+
+**NOTE:** Installation using operator YAMLs is not the supported way any longer.  
+We can install the latest release of OpenEBS ZFS driver by running the following command:
+```bash
+helm repo add openebs https://openebs.github.io/openebs
+helm repo update
+helm install openebs --namespace openebs openebs/openebs --create-namespace
+```
+
+**NOTE:** If you are running a custom Kubelet location, or a Kubernetes distribution that uses a custom Kubelet location, the `kubelet` directory must be changed on the helm values at install-time using the flag option `--set zfs-localpv.zfsNode.kubeletDir=<your-directory-path>` in the `helm install` command.
+
+- For `microk8s`, we need to change the kubelet directory to `/var/snap/microk8s/common/var/lib/kubelet/`, we need to replace `/var/lib/kubelet/` with `/var/snap/microk8s/common/var/lib/kubelet/`.
+- For `k0s`, the default directory (`/var/lib/kubelet`) should be changed to `/var/lib/k0s/kubelet`.
+- For `RancherOS`, the default directory (`/var/lib/kubelet`) should be changed to `/opt/rke/var/lib/kubelet`.
+
+Verify that the ZFS driver Components are installed and running using below command. Depending on number of nodes, you will see one zfs-controller pod and zfs-node daemonset running on the nodes :
+```bash
+$ kubectl get pods -n openebs -l role=openebs-zfs
+NAME                                              READY   STATUS    RESTARTS   AGE
+openebs-zfs-localpv-controller-f78f7467c-blr7q    5/5     Running   0          11m
+openebs-zfs-localpv-node-h46m5                    2/2     Running   0          11m
+openebs-zfs-localpv-node-svfgq                    2/2     Running   0          11m
+openebs-zfs-localpv-node-wm9ks                    2/2     Running   0          11m
+```
+
+Once ZFS driver is installed and running we can provision a volume.
+
+### Deployment
+
+#### 1. Create a Storage class
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-zfspv
+parameters:
+  recordsize: "128k"
+  compression: "off"
+  dedup: "off"
+  fstype: "zfs"
+  poolname: "zfspv-pool"
+provisioner: zfs.csi.openebs.io
+```
+
+The storage class contains the volume parameters like recordsize(should be power of 2), compression, dedup and fstype. You can select what are all parameters you want. In case, zfs properties paramenters are not provided, the volume will inherit the properties from the ZFS Pool.
```

Comment:
empty space after recordsize

Reply:
In case the ZFS properties are not provided as parameters, the volume will inherit the properties from the ZFS pool or the defaults.

Reply:
Done.